fst/en/inflection.py

```diff
@@ -0,0 +1,100 @@
+"""
+Inflects English nouns - creates a plural form from singular.
+
+Install Pynini package before running.
+"""
+
+import pynini as p
+
+from pynini.lib import pynutil
+from pynini.lib import rewrite
+
+def _priority_union(q: p.Fst, r: p.Fst, sigma: p.Fst) -> p.Fst:
+  complement_domain_q = sigma - p.project(q, 'input')
+  return p.union(q, complement_domain_q @ r)
+
+_v = p.union('a', 'e', 'i', 'o', 'u')
+_c = p.union('b', 'c', 'd', 'f', 'g', 'h', 'j', 'k', 'l', 'm', 'n',
+             'p', 'q', 'r', 's', 't', 'v', 'w', 'x', 'y', 'z')
+_sigma = p.union(_v, _c).closure().optimize()
+
+_exceptions = p.string_map([
+    # Zero plurals.
```

Comment:
I count about 444 words in English that follow this rule so far. This is a good start.